clang/test/Driver/sycl-offload-header-check.cpp

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: system-windows
```

Comment:
Is Linux unaffected/guarded by the existing LITs?

Reply:
Test is not valid on Linux.  The compiler will complain based the TMP env variable pointing to an invalid directory (case mismatch)

Reply:
Ah, I rather meant "is Linux sure to be unaffected by the code changes"

Reply:
Thanks for clarifying.  The code path in question is for Linux as well, but should be well tested with existing integration header checks in sycl-offload.c